xai_components/xai_test_compoments/test_components.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+#
+# "TheVirtualBrain - Widgets" package
+#
+# (c) 2022-2023, TVB Widgets Team
+#
+
+import numpy
+from xai_components.base import InArg, Component, xai_component
+
+
+@xai_component(color='rgb(85, 37, 130)')
+class NumpyTest(Component):
+    numpy_code_str: InArg[numpy.ndarray]
```

Comment:
Should we still keep the test component even after you have tvb components with numpy.ndarray input types?

Reply:
No, I don't think so. I will remove it after I'm done with this task.